packages/cli/src/commands/test.ts

```diff
@@ -29,7 +29,7 @@ const defaultIgnored = [
 const testAbortController = new AbortController()
 
 async function storeTest(tmpDir: string) {
-  const testProcess = spawn('yarn test:e2e', {
+  const testProcess = spawn('na run test:e2e', {
```

Comment:
question: why in this command wasn't used the `getPreferredPackageManager()` like the others?

Reply:
it was outdated. thanks for noticing that out!